File: easy/palindrome-linked-list/palindrome-linked-list.py
class Solution(object):
    def isPalindrome(self, head):
        """
        :type head: ListNode
        :rtype: bool
        """
        arr = []
        while head:
            arr.append(head.val)
            head = head.next
        new_arr = reversed(arr)
        return arr == arr[::-1]

File: easy/palindrome-linked-list/palindrome-linked-list.cpp
class Solution {
public:
    bool isPalindrome(ListNode* head) {
        vector<int> res;
        while (head) {
            res.push_back(head->val);
            head=head->next;
        }
        vector<int> temp=res;
        reverse(begin(temp),end(temp));
        if (res==temp) {
            return true;
        }
        return false;
    }
};
